tests/doc-links.spec.ts

```diff
@@ -0,0 +1,235 @@
+    "core.dns_address",
+  );
+  await validateServerSettingLink(
+    page,
+    "How to expose LXD to the network",
+    "/documentation/howto/server_expose/#server-expose",
+    "core.https_address",
+  );
+  await validateServerSettingLink(
+    page,
+    "How to monitor metrics",
+    "/documentation/metrics/",
+    "core.metrics_address",
+  );
+  await validateServerSettingLink(
+    page,
+    "How to manage storage buckets and keys",
+    "/documentation/howto/storage_buckets/#howto-storage-buckets",
+    "core.storage_buckets_address",
+  );
+  await validateServerSettingLink(
+    page,
+    "Instance placement scriptlet",
+    "/documentation/explanation/clustering/#clustering-instance-placement-scriptlet",
+    "instances.placement.scriptlet",
+  );
+});
+
+test("Ensure the documentation link text and link targets are present: Instance Detail > Configuration", async ({
+  page,
+  lxdVersion,
+}) => {
+  test.skip(
+    lxdVersion === "5.0-edge",
+    "Unavailable API Extensions: metadata_configuration",
+  );
+  const instance = randomInstanceName();
+  await createInstance(page, instance);
+  await editInstance(page, instance);
+
+  await page.getByText("Security policies").click();
+
+  await activateOverride(page, "Privileged (Containers only)");
+  await validateDocumentationLink(
+    page,
+    "Container security",
+    "/documentation/explanation/security/#container-security",
+  );
+  await activateOverride(
+    page,
+    "Allow /dev/lxd in the instance (Containers only)",
+  );
+  await validateDocumentationLink(
+    page,
+    "Communication between instance and host",
+    "/documentation/dev-lxd/",
+  );
+
+  await page.getByLabel("Configuration").getByText("Snapshots").click();
+
+  await activateOverride(page, "Snapshot name pattern");
+  await validateDocumentationLink(
+    page,
+    "Automatic snapshot names",
+    "/documentation/reference/instance_options/#instance-options-snapshots-names",
+  );
+
+  await page.getByText("YAML Configuration").click();
+
+  await validateDocumentationLink(
+    page,
+    "Learn more about instances",
+    "/documentation/instances",
+  );
+
+  // Set Down
+  await deleteInstance(page, instance);
+});
+
+test("Ensure the documentation link text and link targets are present: Project > Configuration", async ({
+  page,
+  lxdVersion,
+}) => {
+  test.skip(
+    lxdVersion === "5.0-edge",
+    "Unavailable API Extensions: metadata_configuration",
+  );
+  const project = randomProjectName();
+  await createProject(page, project);
+  await page.getByRole("link", { name: "Configuration" }).click();
+  await page.getByRole("button", { name: "Edit configuration" }).click();
+  await page.getByText("Allow custom restrictions on a project level").click();
+
+  await page.getByText("Resource limits").click();
+  let buttons = page.getByRole("row").getByRole("button");
+  for (let i = 0; i < (await buttons.count()); i++) {
+    await buttons.nth(i).click();
+  }
```

Comment:
Can we also use the `activateOverride` helper here? I think it works on all configuration pages.